spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -90,47 +95,53 @@ def create_workspace_csv(workspaces, client_id_column)
     prod_connection_parameters = {
       username: @config[:username],
       password: @config[:password],
-      server: "https://#{@config[:prod_server]}"
+      server: "https://#{@config[:prod_server]}",
+      verify_ssl: false
     }
     @prod_rest_client = GoodData.connect(prod_connection_parameters)
     @prod_ads = ConfigurationHelper.create_development_datawarehouse(client: @prod_rest_client,
                                                                      title: 'Production ADS',
                                                                      auth_token: @config[:vertica_prod_token])
-
-    @prod_output_stage_project = ConfigurationHelper.create_output_stage_project(
-      @prod_rest_client,
-      @suffix,
-      @prod_ads,
-      @config[:prod_token],
-      @config[:environment]
-    )
-    production_output_stage_uri = @prod_output_stage_project.add.output_stage.data['schema']
+    unless GoodData::Environment::VCR_ON
+      @prod_output_stage_project = ConfigurationHelper.create_output_stage_project(
+        @prod_rest_client,
+        @suffix,
+        @prod_ads,
+        @config[:prod_token],
+        @config[:environment]
+      )
+      production_output_stage_uri = @prod_output_stage_project.add.output_stage.data['schema']
+    end
 
     segments = (%w(BASIC PREMIUM) * ($segments_multiplier || 1)).map.with_index do |segment, i|
-      {
-        segment_id: "INSURANCE_DEMO_#{segment}_#{i}_#{@suffix}",
+      data = {
+        segment_id: "LCM_SPEC_#{segment}_#{i}",
         development_pid: @project.obj_id,
         driver: segment == 'PREMIUM' ? 'vertica' : 'pg',
-        master_name: "Insurance Demo Master (#{segment} #{i}) " + '##{version}',
-        ads_output_stage_uri: production_output_stage_uri,
-        ads_output_stage_prefix: Support::OUTPUT_STAGE_PREFIX
+        master_name: "LCM spec master project (#{segment} #{i}) " + '##{version}',
       }
+
+      unless GoodData::Environment::VCR_ON
+        data[:ads_output_stage_uri] = production_output_stage_uri
+        data[:ads_output_stage_prefix] = Support::OUTPUT_STAGE_PREFIX
+      end
+      data
     end
     segments_filter = segments.map { |s| s[:segment_id] }
 
     @workspaces = (segments * ($workspaces_multiplier || 2)).map.with_index do |segment, i|
       {
-        client_id: "INSURANCE_DEMO_#{i}_#{@suffix}",
+        client_id: "LCM_SPEC_CLIENT_#{i}",
         segment_id: segment[:segment_id],
-        title: "Insurance Demo Workspace #{i} #{@suffix}"
+        title: "LCM SPEC PROJECT #{i}"
```

Comment:
Make sure you update the new project names in the clean-up script. Or better yet, DRY things up by moving the name to a constant.